scripted/plugin/src/main/scala/sbt/ScriptedPlugin.scala

```diff
@@ -0,0 +1,10 @@
+/*
+ * sbt
+ * Copyright 2011 - 2017, Lightbend, Inc.
+ * Copyright 2008 - 2010, Mark Harrah
+ * Licensed under BSD-3-Clause license (see LICENSE)
+ */
+
+package sbt
+
+// ScriptedPlugin has moved to main.
```

Comment:
Why does this file exist?

Reply:
This is sort of a memo to ourselves.
ScriptedPlugin itself has moved to the mothership, but I am leaving the module as is for backward compatibility of the build. In general having a subproject for the plugin, but _not_ having the plugin defined in it is highly unusual, so I'm leaving the trail blaze here.

Reply:
I see what you mean.